### PR TITLE
Editing videos.

### PR DESCRIPTION
**Pull request by DarryQueen**
Adds paths for administrators to edit videos. Fairly self-explanatory.

Reviewer: @nalnaji @delicioustoast

### CODE REVIEW

**Comment by delicioustoast** [comment]
You can edit a video so that its title is blank, but that's probably fine.

Otherwise LGTM.

File: app/views/videos/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+          </div>
+
+          <%= f.submit 'Update', :class => 'btn btn-info center-block' %>
+        <% end %>
+      </div>
+    </div>
+  </div>
+</div>
+
+
```
Comment (nalnaji) [inline app/views/videos/edit.html.erb]: TOO MANY SPACES

**Comment by nalnaji** [comment]
Wow, what a complicated fix!!! GOOD JOB DARREN!

**Comment by DarryQueen** [comment]
Muito obrigado.

**Pull request merged**